src/RwFramework/GSyst.cxx

```diff
@@ -121,23 +121,9 @@ std::map<GSyst_t, std::string> GSyst::BuildGSystToStringMap() {
   temp_map[ kXSecTwkDial_CoulombCCQE ] = "CoulombCCQE";
   temp_map[ kXSecTwkDial_NormCCCOHpi ] = "NormCCCOH";
   temp_map[ kXSecTwkDial_NormNCCOHpi ] = "NormNCCOH";
-  temp_map[ kXSecTwkDial_ZExpELFF ] = "ZExpELFFCCQE";
```

Comment:
Was it intended that kXSecTwkDial_ZExpELFF* be removed when adding kINukehA2025_cex; or are these being excised by accident from working with an outdated version of the code?